internal/cli/atlas/clusters/create.go

```diff
@@ -81,11 +80,14 @@ func (opts *CreateOpts) Run() error {
 	}
 
 	r, err := opts.store.CreateCluster(cluster)
-	var target *atlas.ErrorResponse
-	if errors.As(err, &target) && target.ErrorCode == "INVALID_ATTRIBUTE" && strings.Contains(target.Detail, "regionName") {
+	apiError, ok := atlasv2.AsError(err);
+	code := apiError.GetErrorCode();
+	if ok && apiError.GetErrorCode() == "INVALID_ATTRIBUTE" && strings.Contains(apiError.GetDetail(), "regionName") {
```

Comment:
In v2, this is the Error struct:
type Error struct {
	Detail *string `json:"detail,omitempty"`
	// HTTP status code returned with this error.
	Error *int `json:"error,omitempty"`
	// Application error code returned with this error.
	ErrorCode *string `json:"errorCode,omitempty"`
	// Parameter uses to give more information about the error.
	Parameters []interface{} `json:"parameters,omitempty"`
	// Application error message returned with this error.
	Reason *string `json:"reason,omitempty"`
}

Meaning `apiError.GetDetail()` will probably give the HTTPCode based on the comment description.

Reply:
```
{"detail":"Invalid attribute regionName specified.","error":400,"errorCode":"INVALID_ATTRIBUTE","parameters":["regionName"],"reason":"Bad Request"}
```

Reply:
Thanks for adding this!